package.json

```diff
@@ -61,6 +61,7 @@
     ]
   },
   "dependencies": {
+    "aria-query": "^5.0.0",
```

Comment:
@judithhinlung Looks like the build is failing because the listed deps are out of alphabetical order. `aria-query` should come after `@lint-todo/utils`.